WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -106,12 +109,13 @@ public async Task<bool> StartRoundAsync(IEnumerable<SmartCoin> smartCoins, Round
 				var registeredCoins = registeredAliceClients.Select(x => x.SmartCoin.Coin);
 				var availableVsize = registeredAliceClients.SelectMany(x => x.IssuedVsizeCredentials).Sum(x => x.Value);
 
-				// Calculate outputs values
+				var amountDecomposer = await amountDecomposerTask;
+
 				roundState = await RoundStatusUpdater.CreateRoundAwaiter(rs => rs.Id == roundState.Id, cancellationToken).ConfigureAwait(false);
 				constructionState = roundState.Assert<ConstructionState>();
-				AmountDecomposer amountDecomposer = new(roundState.FeeRate, roundState.CoinjoinState.Parameters.AllowedOutputAmounts.Min, Constants.P2WPKHOutputSizeInBytes, (int)availableVsize);
```

Comment:
`roundState.CoinjoinState.Parameters.AllowedOutputAmounts.Min` this is never used in the new code.